lldb/include/lldb/lldb-types.h

```diff
@@ -62,12 +62,15 @@ typedef void *thread_arg_t;             // Host thread argument type
 typedef void *thread_result_t;          // Host thread result type
 typedef void *(*thread_func_t)(void *); // Host thread function type
 typedef int pipe_t;                     // Host pipe type
+typedef int destroy_callback_token_t;   // Debugger destroy callback token type
```

Comment:
Should this be unsigned?

Reply:
I liked that the invalid token `LLDB_INVALID_DESTROY_CALLBACK_TOKEN` is -1, similar to how `std::basic_string::npos` is -1.

Technically I feel signed/unsigned are pretty much the same: they both have the same number of valid and invalid values; the positive/negative-ness of the values don't matter.

I don't have a strong opinion. Can change if you still think it will be better. In that case, I'm curious to learn your reasons - I must have missed something.

Reply:
Since we are adding this to the public API, we should remove the "destroy" from this so we could re-use this for other callback tokens. So change this to:
```
typedef int callback_token_t;
```
And remove the comment since making a comment of `// Callback token type` doesn't really tell you anything more than the name already does. The reason being we don't want to have to add a ton of callback token types for every callback we might want to register in the future (`process_callback_token_t`, `thread_callback_token_t`, ...). We can just use one for all of them in the future.